contracts/tests/sbtc-token.test.ts

```diff
@@ -87,5 +87,52 @@ describe("sBTC token contract", () => {
       expect(receipt2.value).toEqual(999n);
     });
 
+    test("Fail a non-protocol principal calling protocol-mint", () => {
+      const receipt = txErr(
+        token.protocolMint({
+          amount: 1000n,
+          recipient: bob,
+        }),
+        bob
+      );
+      expect(receipt.value).toEqual(token.constants.ERR_NOT_AUTH.value);
```

Comment:
Nit: I added a little errors helper which cleans up this code. First, in `helpers.ts` line 39 https://github.com/stacks-network/sbtc/blob/bcd04eb792e1886669aa21e5c524a93ea5d9c914/contracts/tests/helpers.ts#L39

add a new line: 

```ts
token: _errors.sbtcToken,
```

And then you can change this line to:

```ts
expect(receipt.value).toEqual(errors.token.ERR_NOT_OWNER);
```

Reply:
Makes sense, updated.